components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -15,8 +15,8 @@
 #include "base/files/file_path.h"
 #include "base/logging.h"
 #include "base/metrics/histogram_macros.h"
+#include "bat/ads/ads_util.h"
 #include "bat/ads/pref_names.h"
```

Comment:
pref_names.h should go in a separate target so using prefs does not require a dep on anything else in bat/ads

Reply:
also I see `bat/ads` includes added in places that likely need deps added

Reply:
For this PR so it does not get out of hand I have only fixed deps for related changes